### ISSUE TITLE
시스템16 성능 실험

### ISSUE THREAD

**Issue by februa22** [nlp-research/biaffineparser#57]

**Comment by februa22** [comment]
데이터셋 및 임베딩 normalize 적용:

- 숫자
- 한자
- 외국어

나머지는 [시스템6](
https://github.com/nlp-research/biaffineparser/issues/42)과 동일함.

**Comment by februa22** [comment]
성능:

> epoch: 7 - loss: 0.2821555733680725 - uas: 0.9212185144424438 - las: 0.9002100825309753 - eval_loss: 0.38095930218696594 - eval_uas: 0.9161061644554138 - eval_las: 0.8917040824890137

**Comment by februa22** [comment]
```json
{
    "num_lstm_units": 400,
    "num_lstm_layers": 3,
    "arc_mlp_units": 500,
    "label_mlp_units": 100,
    "word_embed_size": 200,
    "pos_embed_size": 100,
    "optimizer": "adam",
    "learning_rate": 0.001,
    "decay_factor": 0.9,
    "num_train_epochs": 100,
    "train_filename": "data/sejong.train.csv",
    "dev_filename": "data/sejong.test.csv",
    "out_dir": "parser/model_iss57",
    "word_vocab_name": "word.pkl",
    "pos_vocab_name": "pos.pkl",
    "rel_vocab_name": "rel.pkl",
    "head_vocab_name": "head.pkl",
    "word_embed_file": "embeddings/words.pos.normalize.vec",
    "pos_embed_file": "embeddings/words.tag.original.vec",
    "word_embed_matrix_file": "embeddings/word_embed_matrix.txt",
    "pos_embed_matrix_file": "embeddings/pos_embed_matrix.txt",
    "embed_dropout": 0.33,
    "lstm_dropout": 0.33,
    "mlp_dropout": 0.33,
    "batch_size": 128,
    "device": "gpu",
    "debug": false
}
```
